### PR TITLE
Network upgrade

### PR DESCRIPTION
**Pull request by dennis-school**
A proposal for network upgrade for both Attribution and Profiling.

Don't merge this until it is accepted and implemented in both frontend and backend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,22 +10,68 @@
...
+
+{-| Response from server
+
+Example JSON:
+{ "age": "20-30"
+, "gender": "M" }
+
+-}
+type alias FromServerProfiling =
+    { age: String, gender: String }
```
Comment (folkertdev) [inline README.md:75]: can we represent age as a tuple (or, in TS, fixed-size array)? I like that better

Comment (dennis-school) [reply README.md:75]: Maybe. I have no idea what the Profiling system returns for this, and whether an age range even makes sense.
String is currently just a placeholder.

Comment (folkertdev) [reply README.md:75]: I'll leave this unimplemented for now then. 

The file upload has now been merged into development. The definition of en- and decoders is in `Types.elm`.

**Comment by dennis-school** [comment]
Is the profiling frontend supposed to do anything with the received message?

Otherwise, I think the new network protocol is fully implemented on the webserver.
And should be fully functional.

**Comment by folkertdev** [comment]
not really. There is nothing sensible we can do for now.

**Review by folkertdev — APPROVED** [review]

**Pull request merged**